_inc/client/components/settings-card/index.jsx

```diff
@@ -26,35 +27,42 @@ import Banner from 'components/banner';
 import Button from 'components/button';
 
 export const SettingsCard = props => {
-	let module = props.module
+	const module = props.module
 			? props.getModule( props.module )
-			: false,
-		header = props.header
-			? props.header
-			: '',
-		isSaving = props.isSavingAnyOption(),
+			: false;
+
+	// Non admin users only get Publicize, After the Deadline, and Post by Email settings. The UI doesn't have settings for Publicize.
+	// composing is not a module slug but it's used so the Composing card is rendered to show AtD.
+	if ( ! props.userCanManageModules && ! includes( [ 'composing', 'post-by-email' ], props.module ) ) {
+		return <span />;
```

Comment:
To avoid unneeded elements let's `return null` here, this improves processing time as well because there's no need to create an empty element.